Sources/SentryCrash/Recording/Monitors/SentryCrashMonitor_System.m

```diff
@@ -469,6 +470,19 @@
     return [[NSBundle mainBundle] pathForResource:@"embedded" ofType:@"mobileprovision"] != nil;
 }
 
+/**
+ * Check if the embdded.mobileprovision provisions all devices
+ * If true the profile is for `enterprise`, if not it is `adhoc`
+ */
+static bool
+mobileProvisionProfileProvisionsAllDevices(void)
```

Comment:
`m`: I think this method could be part of the `SentryMobileProvisionParser`. All the `SentryMobileProvisionParser` needs to do is to check if there is an embedded mobile provision and if yes which one.